#To overcome compilation issues like:
#  error: 'unique_ptr' is not a member of 'std'
#  error: expected primary-expression before '[' token

--- ark-18.04.0/plugins/libzipplugin/libzipplugin.cpp.orig	2018-04-09 22:42:03.000000000 +0200
+++ ark-18.04.0/plugins/libzipplugin/libzipplugin.cpp	2018-04-15 12:41:56.490025275 +0200
@@ -39,6 +39,8 @@
 #include <qplatformdefs.h>
 #include <QThread>
 
+#include <memory>
+
 #include <utime.h>
 #include <zlib.h>
 
